ci/get-test-half.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# get_test_half returns either the first or second half of integration tests
+# Usage: get_test_half <half_number>
+# half_number: 0 for first half, 1 for second half
+get_test_half() {
```

Comment:
Making separate function with file looks cleaner from maintenance perspective. Can we make this function a bit abstracted in a way to split specs into N chunks?
If it is something initial version, LGTM!